config/src/main/java/com/typesafe/config/ConfigValueFactory.java

```diff
@@ -60,10 +65,12 @@ private ConfigValueFactory() {
      *            object to convert to ConfigValue
      * @param originDescription
      *            name of origin file or brief description of what the value is
+     * @param comments
+     *            comments on this ConfigValue used when it's rendered
```

Comment:
this should explain why comments is a list instead of a single string. In fact, should it be a single string? It could have newlines in it.